src/lib/PostPage.svelte

```diff
@@ -291,10 +316,6 @@
     cursor: pointer;
   }
 
-  .credit-line {
```

Comment:
You were wondering why this is here when it is not getting used.
I had implemented the credits before and this was part of it - since we couldn't agree on how it should be displayed I removed it - this is what is left. Removing it is sensible since it won't be useful :+1: